tests/Integration/Classes/ObjectModelTest.php

```diff
@@ -94,6 +94,18 @@ protected function setUp(): void
         $this->secondShopId = Shop::getIdByName('Shop 2');
     }
 
+    public function testTrans(): void
```

Comment:
I consider a good practice to describe the purpose of this test. Without context, it's very difficult to understand what you are testing here.